OpenRA.Game/Actor.cs

```diff
@@ -58,6 +58,7 @@ public Activity CurrentActivity
 		public IEffectiveOwner EffectiveOwner { get; private set; }
 		public IOccupySpace OccupiesSpace { get; private set; }
 		public ITargetable[] Targetables { get; private set; }
+		public IResolveOrder[] ResolveOrderTraits { get; private set; }
```

Comment:
We don't include `Traits` in the other names, so can we please rename this to hust `ResolveOrders`?

Reply:
IMO it would also be cleaner if we added an a `public void ResolveOrder(Order order)` method that loops over these, so they can stay private.